jupyter_book/utils.py

```diff
@@ -171,3 +171,11 @@ def _check_book_versions(path_book):
         )
 
     return True
+
+
+def _list_files_with_extension(lookup_folder, extension=".ipynb"):
+    """ fetch the list of files in the specified folder """
+    for root, dir, files in os.walk(lookup_folder):
```

Comment:
Will this walk into hidden `.ipynb_checkpoints` directories? Is it desirable to go into such folders and rewrite those notebooks?